### ISSUE TITLE
IdsEditor: Accessibility WCAG Guideline: 1.4.13 Content on Hover or Focus Defect

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1318]
**Describe the bug**
Tooltip is not provided for 3 dot icon.

To Reproduce
"1. Launch the URL: https://main.wc.design.infor.com/ids-dropdown/example.html
2. Navigate to 3 dot icon and verify"
3. Tooltip is not provided for 3 dot icon.

Expected behavior : Tooltip should be provided for 3 dot icon.

User Impact:  If tooltip will not be provided then user will not be able to identify the icons.

Screenshots If applicable, add screenshots to help explain your problem: https://www.screencast.com/t/cdiI3CQw

Dev Recommendation: "1. Create a <div> element, give it a role=""tooltip"" and an id=""IDREF"". Then, give it a brief description of the action.
2. This element has to be accessible by focusing as well as hovering over. Both functionalities can be provided using CSS.
3. The tooltip has to be visually hidden using CSS.
4. Associate the tooltip with its button using aria-labelledby.
5. After applying this keyboard-accessible tooltip, consider removing the title attribute from the element.

Refer to:
https://inclusive-components.design/tooltips-toggletips/
https://a11yproject.com/posts/how-to-hide-content/"

Platform
* Windows 10/Chrome/NVDA

Additional context
NA

**Comment by tmcconechy** [comment]
Yes we could add a tooltip.
